src/api/python/wheels/build_wheel.py

```diff
@@ -0,0 +1,150 @@
+###############################################################################
+# Top contributors (to current version):
+#   Makai Mann
+#
+# This file is part of the cvc5 project.
+#
+# Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+# in the top-level source directory and their institutional affiliations.
+# All rights reserved.  See the file COPYING in the top-level source
+# directory for licensing information.
+# ############################################################################
+#
+# Script for building wheels distribution for pycvc5
+#
+# Example usage (from top-level directory):
+#   python3 ./src/api/python/wheels/build_wheel.py bdist_wheel
+# Configures and builds in directory ./build
+# Creates wheel in ./dist
+#
+##
+
+import os
+import re
+import sys
+import platform
+import subprocess
+import multiprocessing
+import shutil
+import glob
```

Comment:
Seems to be unused.

Reply:
Thanks, fixed with 5e08ae1f2f2ec4602e451fc9670ae7970cf031b1.